modules/configure-cvo-retrieve-image-signatures.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+// * updating/updating_a_cluster/updating_disconnected_cluster/disconnected-update-osus.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="configure-cvo-retrieve-image-signatures_{context}"]
+= Configuring the Cluster Version Operator (CVO) to retrieve release image signatures
+
+If you are using the oc-mirror plugin 4.15 or later and the OSUS Operator 5.0.3 or later to update your cluster in a disconnected environment, you can configure the Cluster Version Operator (CVO) to retrieve release image signatures from an endpoint on the local OpenShift Update Service (OSUS).
```

Comment:
> from an endpoint on the local OpenShift Update Service (OSUS).

I'm not sure about the "local". Our recommended topology is to have a single OSUS deployed somewhere (on one OCP cluster) and and let more (many!) OCP clusters consume its services. In technical context I'd read "local" as "running in the same cluster" which is not necessary and I'm worried that it reinforces the anti-pattern that you should run one OSUS inside every cluster.

I guess "local" can also mean "local to your organization; not RH-operated" but I'd still avoid call that "local" because of the possible confusion.

Reply:
If you look at the [rest of the page](https://docs.openshift.com/container-platform/4.15/updating/updating_a_cluster/updating_disconnected_cluster/disconnected-update-osus.html) that this content is being added to and search for "local", you'll find many instances where we use this "local OSUS" wording. 

I don't disagree with implementing this change generally and in this PR, but I think that if we do try to shy away from this wording, we'd want to make a broader sweep throughout this page to make the change effective. 

Would it also be better to find another word to convey "not the OSUS hosted by RH"? I guess without it, the context of the page is probably enough to inform users, but maybe clarity wherever possible might be a safer bet.

What about this wording?
```suggestion
If you are using the oc-mirror plugin 4.15 or later and the OSUS Operator 5.0.3 or later to update your cluster in a disconnected environment, you can configure the Cluster Version Operator (CVO) to retrieve release image signatures from an endpoint on the OpenShift Update Service (OSUS) in your environment.
```

Reply:
Good point about consistency with other instances. I haven't searched but I agree that if we use it in more places then doing it in a separate, broad PR is preferable over changing a single instance in this PR :+1:

"in your environment" sounds good to me

Reply:
Sounds good! I guess I'll keep "local" in this PR and then I can create another Jira issue to cover the broader change. That'll also give us all more time to discuss possible wording options in addition to "in your environment"

Reply:
Since I'm starting the work to update the "local" wording before this PR is finished, I'll go ahead and update this one instance here.